### DISCUSSION TITLE
채팅방을 퇴장했다는 메시지 없이 채팅방 퇴장하기

### DISCUSSION THREAD

**Discussion by minmoong** [storycraft/node-kakao#549] [Q&A]
를 하기위해 어떤 메소드를 사용해야하나요?
퇴장 이벤트가 발생되었을때 퇴장 패킷을 드랍시켜야 하나요?

**Comment by generalxd** [comment]
불가능

**Reply by minmoong** [reply]
찾아보니까 있더라고요 ㄷㄷㄷㄷ
![캡처3](https://user-images.githubusercontent.com/62737839/129480036-da04646e-b959-4141-8a76-19de97c639e2.PNG)
https://devtalk.kakao.com/t/feat-loco-protocol/105350
참고하시길..

**Reply by generalxd** [reply]
퇴장메시지 드랍이아니라 저형식으로 메시지를 쏘는겁니다.

**Reply by minmoong** [reply]
저 형식의 메시지타입이 있나요?

**Reply by generalxd** [reply]
Feed

**Reply by minmoong** [reply]
막혔다는..

**Reply by kiwioverflow** [reply]
저건 막힌지 오래입니다.
Feed로 전송은 더이상 불가능합니다

**Reply by DeltaLaboratory** [reply]
이미 패치되었고, 메시지 없이 퇴장하는건 불가능합니다.
